charts/nri-statsd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.0.0
 description: A Helm chart to deploy the New Relic Statsd integration
 name: nri-statsd
-version: 1.0.0
+version: 1.0.2
```

Comment:
🎗️  depends on https://github.com/newrelic/helm-charts/pull/372/files#diff-28e77334f2e9a2a31350c1e5e7f801593623f59a055eb4276bcf7c3a4c66b729R5

Reply:
Update the version.